joblib/test/test_parallel.py

```diff
@@ -940,3 +940,42 @@ def __reduce__(self):
     with warns(DeprecationWarning):
         with raises(ValueError):
             delayed(UnpicklableCallable(), check_pickle=True)
+
+
+def test_backend_reset():
+    """Test that a backend is correctly reset to its initial state."""
+
+    # let's tolerate 30% error between compuration times to avoid random
+    # failures because of variable available ressources on CIs.
+    error = 0.3
+    n_jobs = 4
```

Comment:
I think we have only 2 cores on Travis, double-check and use n_jobs=2 if that's the case.

Reply:
I tried to use `mp.cpu_count()` but it was worse. Setting 2 jobs is ok.